### ISSUE TITLE
Output example html file as a mustache file

### ISSUE THREAD

**Issue by hbuchel** [jkphl/svg-sprite#119]
Is there a way to make the output file for an example have the .mustache file extension instead of .html?

Here is my sprite config

```
var spriteConfig = {
    shape: {
        spacing: {
            padding: 2
        }
    },
    dest: '',
    mode: {
        css: {
            bust: false,
            prefix: '.icon-%s',
            dest: 'public',
            sprite: 'images/sprites.svg',
            example: {
              dest: '../source/_patterns/00-base/03-images/01-icons.mustache',
              template: './source/images/svg-sprites/template.html'
            },
            render: {
                scss: {
                    dest: '../source/scss/base/_sprites.scss',
                    template: 'source/images/svg-sprites/sprite.scss'
                }
            }
        }
    }
};
```

For my example html document that gets output, I'm using a custom template (needed to change the background of the .icon-box class amongst some other change) but I would like the output file (dest) to be a .mustache file, because then my styleguide generator (I'm using the node version of patternlab) I'm using can pick that up.

Any thoughts?

**Comment by jkphl** [comment]
Hey @hbuchel,

unfortunately it looks like I'm [enforcing an `.html` file extension](https://github.com/jkphl/svg-sprite/blob/master/lib/svg-sprite/mode/base.js#L130) on the example output file at the moment. It's probably OK to loosen this restriction. I'll try to push out another release tonight, so your code should just work as you expect.

Cheers, Joschi

**Comment by jkphl** [comment]
@hbuchel Should be fixed with [release v1.2.12](https://github.com/jkphl/svg-sprite/releases/tag/v1.2.12)
